src/metaImage.cxx

```diff
@@ -683,27 +683,27 @@ ElementSizeValid(bool _elementSizeValid)
   m_ElementSizeValid = _elementSizeValid;
   }
 
-const float * MetaImage::
+const double * MetaImage::
 ElementSize(void) const
   {
   return m_ElementSize;
   }
 
-float MetaImage::
+double MetaImage::
 ElementSize(int _i) const
   {
   return m_ElementSize[_i];
   }
 
 void MetaImage::
-ElementSize(const float *_elementSize)
+ElementSize(const double *_elementSize)
   {
-  memcpy(m_ElementSize, _elementSize, m_NDims*sizeof(float));
+  memcpy(m_ElementSize, _elementSize, m_NDims*sizeof *m_ElementSize);
```

Comment:
should it be `m_NDims*sizeof(m_ElementSize)`

Reply:
I want m_NDims* sizeof(double) here

Reply:
gotcha ... then should we introduce ElementSizeType ?

Reply:
Well the other type (offset, transform) are implicitly declared as `double`, since we only need the `sizeof` here (and not `typeof`), I believe we can keep the change small.

Reply:
Agreed. After switching to using parenthesis with `sizeof` 👍

Reply:
Does the ITK/VTK/MetaIO coding style enforce usage of parenthesis (non type-name case here) ?